stackit/internal/conversion/conversion.go

```diff
@@ -58,6 +59,22 @@ func ToTerraformStringMap(ctx context.Context, m map[string]string) (basetypes.M
 	return res, nil
 }
 
+// FromTerraformStringMapToInterfaceMap converts a basetypes.MapValue of Strings to a map[string]interface{}.
+func FromTerraformStringMapToInterfaceMap(ctx context.Context, m basetypes.MapValue) (map[string]interface{}, error) {
```

Comment:
[Nitpick] Isn't `ToStringInterfaceMap` (or something similar) enough for the name? We can already tell by the signature it is from a TF map

Reply:
I was trying to match the naming of the `ToTerraformStringMap`, but you are right, it doesn't have the `From` part